### PR TITLE
Create pull request templates

### PR DESCRIPTION
**Pull request by Erin-Cecele**

### CODE REVIEW

**Comment by Erin-Cecele** [comment]
@geealbers I made changes to the bug fix template that are ready for review. 

One note: I removed "If this is a new bug, please be sure to post it on our [issue tracker](https://github.com/thegetty/quire/issues/new?assignees=erin-cecele%2Cgeealbers&labels=bug&template=bug_report.yaml_) first" because I believe we decided it was pointless if they were already at the PR stage.

 I am still working on the "general" template that replaces the feature request one.

**Comment by Erin-Cecele** [comment]
@geealbers I also just updated the "feature request" PR to a "general" PR. I tried to differentiate it from the bug-fix one as much as possible. Let me know if you think it works.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_bugfix.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Checklist 
+
+Put an `x` next to the boxes that apply.
+
+- [ ] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file
+
+- [ ] I have made my changes in a new branch and not directly in the main branch
+
+- [ ] I have added tests that prove my fix is effective
```
Comment (geealbers) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_bugfix.md:11]: Did we talk about this? Do we want to mention testing when the team hasn't really defined how it works now or how someone would go about adding tests like this?

**Review by geealbers — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_bugfix.md (hunk 1 above)
Comment (Erin-Cecele) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_bugfix.md:11]: @geealbers I'll just make a note to add to the pull request templates once we have those details finalized.

**Comment by Erin-Cecele** [comment]
@geealbers hmm, I saw you approved this but it needs to be approved again for me to merge.

**Review by geealbers — APPROVED** [review]

**Pull request merged**